src/routes/users.route.ts

```diff
@@ -0,0 +1,11 @@
+import {createUser, getUsers, getUser, deleteUser, updateUser} from "../controllers/users.controller";
+import {FastifyInstance, FastifyServerOptions} from "fastify";
+
+export async function routes (fastify: FastifyInstance, options: FastifyServerOptions) {
+    fastify.post('/create', createUser)

```

Comment:
remove create word from route

just left it like fastify.post('/', createUser)